Syllabus/_child_assignments.Rmd

```diff
@@ -0,0 +1,32 @@
+<!-- This document contains assignment details that appear multiple places. -->
+
+## Grading
+***
+
+1. Attendance/Participation: 20% (Please let the instructors know if attendance will be difficult for you.)
+2. Homework: 3 x 15%
+3. Final "Project": 35%
+
+All assignments are due Wednesday, Jan 26, 2022 at 11:59pm EST.
+
+**Note**: only people taking the course for credit must turn in the assignments. However, we will evaluate all submitted assignments in case others would like feedback on their work.
+
+
+## Submitting Assignments
+***
+
+Submit each assignment to the designated [Drop Box on CoursePlus](https://courseplus.jhu.edu/core/index.cfm/go/db:main/coID/16733/).
+
+You should complete the following:
+
+1. [Data Quest: Introduction to Programming in R](https://app.dataquest.io/m/499/introduction-to-programming-in-r)
+2. Homework 2 Problem Set: HTML, Rmd, key (HTML), key (code), zip download
```

Comment:
Is this a list of items required to be turned in for the problem set? Are the details about them described elsewhere (I haven't looked yet). 
```suggestion
2. Homework 2 Problem Set: 
  - HTML
  - Rmd
  - key (HTML)
  - key (code)
  - zip download
```

Reply:
It's the questions (in two forms- html and Rmd) and the key (also in the same two forms) - we could parse it like that/label it more

Reply:
I think that would help the students

Reply:
Made this more clear - thanks!